Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -460,6 +460,9 @@ async def cancel_make_transport():
             test_utils.run_briefly(self.loop)
 
     def test_close_kill_running(self):
+        if isinstance(asyncio.get_child_watcher(),
+                      unix_events.MultiLoopChildWatcher):
+            raise unittest.SkipTest("Temporary skip until bpo-38323 is fixed")
```

Comment:
```suggestion
            self.skip("Temporary skip until bpo-38323 is fixed")
```

Reply:
Although that would be quite convenient and look better, it unfortunately doesn't work like that. I tested it locally to verify:

```
======================================================================
ERROR: test_close_kill_running (test.test_asyncio.test_subprocess.SubprocessMultiLoopWatcherTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/aeros/repos/aeros-cpython/Lib/test/test_asyncio/test_subprocess.py", line 465, in test_close_kill_running
    self.skip("Temporary skip until bpo-38323 is fixed")
AttributeError: 'SubprocessMultiLoopWatcherTests' object has no attribute 'skip'
```

Reply:
Sorry, it seems like I misremembered the syntax:
```suggestion
            self.skipTest("Temporary skip until bpo-38323 is fixed")
```

Reply:
Oh, okay. That works fine. I'm completely used to using the decorator when possible (it works in 99% cases), so I've personally not used it like that. Thanks for the suggestion, it looks better than `raise SkipTest()`. :-)